docker/launch-netbox.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+load_configuration() {
+  UNIT_CONFIG="/etc/unit/nginx-unit.json"
+
+  MAX_WAIT=10
+  WAIT_COUNT=0
+  while ! nc -z localhost 5555; do
+    if [ $WAIT_COUNT -gt $MAX_WAIT ]; then
+      echo "⚠️ No control socket found; configuration will not be loaded."
+      return 1
+    fi
+
+    WAIT_COUNT=$((WAIT_COUNT + 1))
+    echo "⏳ Waiting for control socket to be created... (${WAIT_COUNT}/${MAX_WAIT})"
+
+    sleep 1
+  done
+
+  # even when the control socket exists, it does not mean unit has finished initialisation
+  # this curl call will get a reply once unit is fully launched
+  curl --silent --output /dev/null --request GET http://localhost:5555/
+
+  echo "⚙️ Applying configuration from $UNIT_CONFIG";
+
+  RESP_CODE=$(curl \
+                  --silent \
+                  --output /dev/null \
+                  --write-out '%{http_code}' \
+                  --request PUT \
+                  --data-binary @$UNIT_CONFIG \
```

Comment:
This should prevent accidental splitting of the parameter into two, e.g. when $UNIT_CONFIG is set to a path containing a whitespace.

```suggestion
                  --data-binary "@${UNIT_CONFIG}" \
```